test.sh

```diff
@@ -1,43 +1,24 @@
 #!/bin/bash -ex
 
-test_image=api-test
-docker build -t $test_image .
-
 function finish {
-	docker rm -f $pg_cid
-	docker rm -f $server_cid
+  docker-compose down
 }
 trap finish EXIT
 
-possum_tag=0.1.0-stable
-possum_image=registry.tld/possum:$possum_tag
-
-export POSSUM_DATA_KEY="$(docker run --rm $possum_image data-key generate)"
-
-pg_cid=$(docker run -d postgres:9.3)
-
-server_cid=$(docker run -d \
-	--link $pg_cid:pg \
-	-e DATABASE_URL=postgres://postgres@pg/postgres \
-	-e RAILS_ENV=test \
-	$possum_image server)
+# Generate reports folders locally
+mkdir -p spec/reports features/reports
 
-set -o pipefail
-admin_api_key=( $(cat ci/setup-account.sh | docker exec -i $server_cid /bin/bash | tail -1) )
+# Build test container & start the cluster
+docker-compose build
+docker-compose up -d
 
-mkdir -p spec/reports features/reports
+# Delay to allow time for Possum to come up
+# TODO: remove this once we have HEALTHCHECK in place
+sleep 20
```

Comment:
I'm still going to give you crap about this. ;)

I don't think we should have a sleep here until with get HEALTHCHECK set up. We should add the `wait_for_server` script from `cli-ruby` and use it, instead. It's still a stopgap, but at least it fails properly when the server doesn't come up in time. This will just continue on and blow up weirdly later on.